requirements/dev.txt

```diff
@@ -10,4 +10,4 @@ onnx==1.15.0
 onnxruntime==1.14.1
 pytest-csv==3.0.*
 tox==4.11.*
-mlflow==2.9.*
+mlflow==2.10.2
```

Comment:
Is there a reason why setting it to `2.10.2` instead of `2.10.*`?

Reply:
@eunwoosh all dependency should be pinned with the exact version. this is some of the guideline of OSSF and the update of dependencies would be done by dependabot.